### PR TITLE
[feature] Authentication backend

### PR DESCRIPTION
**Pull request by raghav-kukreti**
- Minor style changes
- Djoser authentication setup with tokenAuthentication.
- Custom user model, manager and serializer defined.
- Djoser authentication works with custom user model.
- Run `python3 manage.py migrate` before tests

### CODE REVIEW

File: client/src/routes/index.svelte [hunk 1]
```diff
@@ -18,7 +58,7 @@
 <section class="main">
 	<div class="wrapper">
 		<h1>WebMP: Mentor growth</h1>
-		<p>Some small description to make people feel inspired and attracted towards the service.</p>
+		<p>Some small description to make people feel inspired and attracted towards the service. Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum qui itaque omnis ut ex alias ducimus.</p>
```
Comment (ghost) [inline client/src/routes/index.svelte:61]: Let's remove the Latin and put some text of our own here

**Review by ghost — APPROVED** [review]

File: client/src/routes/index.svelte (hunk 1 above)
Comment (raghav-kukreti) [reply client/src/routes/index.svelte:61]: Sure Nivedit, great suggestion!

Comment (vidit17121) [inline client/src/routes/index.svelte:61]: nice.

**Review by vidit17121 — APPROVED** [review]
nice.

**Pull request merged**